src/components/ModalsComponents/ProfileModals/ModalUserInfo/StepPersonal/StepPersonal.jsx

```diff
@@ -37,8 +39,20 @@ const StepPersonal = () => {
       city: city,
       subscribed: userData.subscribed,
       description: description,
-    });
+    }).unwrap();
+
+    const initialValues = {
+      firstName: response.firstName || '',
+      lastName: response.lastName || '',
+      city: response.city || '',
+      country: response.country || '',
+      status: response.status || '',
+      description: response.description || '',
+    };
```

Comment:
```suggestion
    const initialValues = {
      firstName: response.firstName || '',
      lastName: response.lastName || '',
      city: response.city || '',
      country: response.country || '',
      status: response.status || '',
      description: response.description || '',
    };
```

Тут обговрювали додати скелетон, для того щоб не пригали дані в формі